examples/react-native/package.json

```diff
@@ -13,14 +13,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "@aws-amplify/react-native": "console-preview",
     "@aws-amplify/ui": "*",
     "@aws-amplify/ui-react-core": "*",
     "@aws-amplify/ui-react-core-notifications": "*",
     "@aws-amplify/ui-react-native": "*",
     "@react-native-async-storage/async-storage": "^1.17.5",
     "@react-native-community/netinfo": "^8.3.0",
     "@xstate/react": "3.0.0",
-    "amazon-cognito-identity-js": "^5.2.11",
```

Comment:
This is now used in the `ui` package, so added dependency there, also was getting duplicate values errors between this package and `@aws-amplify/react-native` when building the app.